indra/assemblers/indranet/assembler.py

```diff
@@ -48,10 +53,93 @@ def add_statements(self, stmts):
         """
         self.statements += stmts
 
-    def make_model(self, exclude_stmts=None, complex_members=3,
+    def make_model(self, method='df', exclude_stmts=None, complex_members=3,
                    graph_type='multi_graph', sign_dict=None,
-                   belief_flattening=None, weight_flattening=None,
-                   extra_columns=None):
+                   belief_flattening=None, belief_scorer=None,
+                   weight_flattening=None, extra_columns=None):
+        """Assemble an IndraNet graph object.
+
+        Parameters
+        ----------
+        method : str
+            Method for assembling an IndraNet graph. Accepted values: `df` and
+            `preassembly`.
```

Comment:
Could you add a couple more sentences here to explain the difference here for posterity?